corehq/apps/analytics/tasks.py

```diff
@@ -383,6 +383,13 @@ def _log_response(data, response):
         response_text = json.dumps(response.json(), indent=2, sort_keys=True)
     except Exception:
         response_text = response.status_code
-    logger.debug('Sent this data to HS: %s \nreceived: %s' %
-                 (json.dumps(data, indent=2, sort_keys=True),
-                  response_text))
+
+    message = 'Sent this data to HS: %s \nreceived: %s' % (
+        json.dumps(data, indent=2, sort_keys=True),
+        response_text
+    )
+
+    if response.status_code != 200:
+        logger.error(message)
```

Comment:
FYI @calellowitz Currently most of analytics logs are missing, as the logger defined here points to `analytics` logger which no longer exists. So these logs go to the empty default logger which has a level of `INFO` hence all the `DEBUG` levels being triggered in analytics won't get propagated to default django log files. Long term, it might be good to setup logging for analytics. but for now I have made this use `ERROR` level so that all these failures are captured. This will let us investigate why some Hubspot submissions are failing